sdcm/nemesis/monkey/data_operations.py

```
@@ -0,0 +1,700 @@
+        self.verify_using_timestamp_deletions(ks_cf=ks_cf, verification_queries=verification_queries)
+
+        return partitions_for_delete
+
+    def verify_using_timestamp_deletions(self, ks_cf, verification_queries):
+        """Verify that rows deleted via USING TIMESTAMP are actually gone."""
+        mv_not_configured = False
+        mv_table_name = ".".join([ks_cf.split(sep=".")[0], "view_test"])
+        with self.runner.cluster.cql_connection_patient(self.runner.target_node, connect_timeout=300) as session:
+            for pk, ck, ts in verification_queries:
+                result = session.execute(
+                    SimpleStatement(f"SELECT pk, ck, writetime(v) FROM {ks_cf} WHERE pk = {pk} AND ck = {ck}")
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)